rm-datasource/src/main/java/io/seata/rm/datasource/exec/SelectForUpdateExecutor.java

```diff
@@ -113,7 +111,13 @@ public T doExecute(Object... args) throws Throwable {
                     } else {
                         conn.rollback();
                     }
-                    lockRetryController.sleep(lce);
+
+                    // retry hear only when the config is true
+                    if (LockRetryPolicy.isLockRetryPolicyBranchRollbackOnConflict()) {
```

Comment:
it's not consistent with the previous meaning

Reply:
change

Reply:
change